### DISCUSSION TITLE
指定其它端口后显示无网络连接

### DISCUSSION THREAD

**Discussion by wxdbb** [1715173329/UnblockNeteaseMusic#699] [Q&A] [answered]
node app.js 会显示http 和https 的代理 可以正常使用
node app.js -p 2345 只显示http的代理 测试成功但会无网络连接

**Comment by 1715173329** [comment, marked as answer]
-p <http>:<https>
e.g. `-p 2345:2346`

**Comment by wxdbb** [comment]
感谢！
